### PR TITLE
chore: rename RHTAP references to Konflux

### PR DESCRIPTION
**Pull request by tommartensen**
## Description

- RHTAP was renamed to Konflux
- remove libinsp wrapper cleanup according to https://github.com/stackrox/collector/pull/1441#discussion_r1484450706 

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Updated documentation accordingly

**Automated testing**
  - [ ] Added unit tests
  - [ ] Added integration tests
  - [ ] Added regression tests

If any of these don't apply, please comment below.

## Testing Performed

TODO(replace-me)
Use this space to explain how you tested your PR, or, if you didn't test it, why you did not do so. (Valid reasons include "CI is sufficient" or "No testable changes")
In addition to reviewing your code, reviewers **must** also review your testing instructions, and make sure they are sufficient.

For more details, ref the [Confluence page](https://stack-rox.atlassian.net/wiki/spaces/StackRox/pages/855998488/Proposal+Explicitly+List+Testing+Steps+on+PRs) about this section.

### CODE REVIEW

File: .tekton/collector-slim-pull-request.yaml
Comment (msugakov) [inline .tekton/collector-slim-pull-request.yaml:25]: Please adjust it in `collector-slim-push.yaml` too.

**Review by msugakov — APPROVED** [review]

**Comment by tommartensen** [comment]
@msugakov can you approve again? I moved this from "draft" to "ready for review" after your approval and GH does not recognize your approval on behalf of @stackrox/rhtap-maintainers .

**Review by msugakov — APPROVED** [review]

**Review by Stringy — APPROVED** [review]

**Pull request merged**
